sky/serve/replica_managers.py

```diff
@@ -271,6 +306,11 @@ def to_replica_status(self) -> serve_state.ReplicaStatus:
             # Pending to launch
             return serve_state.ReplicaStatus.PENDING
         if self.sky_launch_status == ProcessStatus.RUNNING:
+            if self.sky_down_status == ProcessStatus.RUNNING:
+                # sky.down is running since a scale down interrupted sky.launch
+                return serve_state.ReplicaStatus.SHUTTING_DOWN
+            if self.sky_down_status == ProcessStatus.FAILED:
+                return serve_state.ReplicaStatus.FAILED_CLEANUP
```

Comment:
what if self.sky_down_status is SUCCEEDED

Reply:
added 
```
            if self.sky_down_status == ProcessStatus.SUCCEEDED:
                # This indicate it is a scale_down with correct teardown.
                # Should have been cleaned from the replica table.
                return serve_state.ReplicaStatus.UNKNOWN
```